src/TranslatableTrait.php

```diff
@@ -19,48 +21,13 @@ trait TranslatableTrait
      *
      * @param string      $message    The message to be translated
      * @param array       $parameters Array of parameters used to translate message
-     * @param string|null $domain     The domain for the message or null to use the default
+     * @param string|null $context    The domain for the message or null to use the default
      * @param string|null $locale     The locale or null to use the default
      *
      * @return string The translated string
      */
-    public function _($message, ?array $parameters = null, ?string $domain = null, ?string $locale = null): string
-    {
-        if (isset($this->app) && method_exists($this->app, '_')) {
-            return $this->app->_($message, $parameters, $domain, $locale);
```

Comment:
i think we should keep it so we can user have control of translating process in the app.

(those 3 lines), the rest can fall back to singleton.

Reply:
I more or less second this